### PR TITLE
Change Parcel and Owner to use ApplicationDocuments

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add new doc types

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]
🔥

File: portal-frontend/src/app/features/edit-application/parcel-details/application-owner-dialog/application-owner-dialog.component.ts [hunk 1]
```diff
@@ -72,6 +75,7 @@ export class ApplicationOwnerDialogComponent {
       this.email.setValue(data.existingOwner.email);
       this.existingUuid = data.existingOwner.uuid;
       if (data.existingOwner.corporateSummary) {
+        debugger;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-application/parcel-details/application-owner-dialog/application-owner-dialog.component.ts]: removed

File: portal-frontend/src/app/features/edit-application/parcel-details/parcel-entry/parcel-entry.component.ts [hunk 2]
```diff
@@ -192,18 +193,22 @@ export class ParcelEntryComponent implements OnInit {
...
     if (parcelUuid) {
       const mappedFiles = file.file;
-      await this.applicationParcelService.attachExternalFile(parcelUuid, mappedFiles);
-      this.onFilesUpdated.emit();
+      this.parcel.certificateOfTitle = await this.applicationParcelService.attachCertificateOfTitle(
+        this.fileId,
+        parcelUuid,
+        mappedFiles
+      );
+      debugger;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-application/parcel-details/parcel-entry/parcel-entry.component.ts]: removed

File: portal-frontend/src/app/features/edit-application/parcel-details/parcel-owners/parcel-owners.component.ts [hunk 3]
```diff
@@ -48,8 +49,10 @@ export class ParcelOwnersComponent {
         },
       });
     } else {
+      debugger;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-application/parcel-details/parcel-owners/parcel-owners.component.ts]: removed

**Pull request merged**